tests/commonlib/io_utils.py

```diff
@@ -102,7 +102,7 @@ class FsClient:
     """
 
     @staticmethod
-    def exec_command(
+    def exec_command(  # noqa: C901
```

Comment:
@orestisfl, I agree with flake8 that this function is too complex. Could you please create a task in the Automation infrastructure to refactor the function?

Reply:
https://github.com/elastic/cloudbeat/issues/1003